blas/cblas128/cblas128.go

```diff
@@ -7,12 +7,12 @@ package cblas128 // import "gonum.org/v1/gonum/blas/cblas128"
 
 import (
 	"gonum.org/v1/gonum/blas"
-	"gonum.org/v1/gonum/blas/cgo"
+	"gonum.org/v1/gonum/blas/native"
 )
 
 // TODO(kortschak): Change this and the comment below to native.Implementation
```

Comment:
We need to either remove this comment or make the implementation local to this package and call is something like `no`.